plugins/palo_alto_pan_os/plugin.spec.yaml

```
@@ -453,16 +516,19 @@ actions:
         description: Status of the requested operation e.g. success, error, etc
         type: string
         required: false
+        example: true
```

```suggestion
        example: "success"
```
Not sure, maybe true is already a valid status here?

---

I have updated all of the status to "success"

as this is what we return on a successful request 

```
            "response": {
                "@code": "20",
                "@status": "success",
                "msg": "command succeeded"
            }
 ```